cartography/intel/aws/__init__.py

```diff
@@ -74,18 +73,6 @@ def _sync_one_account(
     if 'resourcegroupstaggingapi' in aws_requested_syncs:
         RESOURCE_FUNCTIONS['resourcegroupstaggingapi'](**sync_args)
 
-    run_analysis_job(
-        'aws_ec2_iaminstanceprofile.json',
-        neo4j_session,
-        common_job_parameters,
-    )
-
-    run_analysis_job(
-        'aws_lambda_ecr.json',
```

Comment:
This one is already a problem for when we want to only sync one account, but we have data from other accounts present in the graph. In such a case, this job would remove data from the other accounts, probably unintended by the user.

Yes, let's remove this invocation from here, but update the job to be more like the `aws_eks_asset_exposure.json`, where we would remove all the `:HAS` relationships, regarldess of `lastupdated`, and then create new `:HAS` relationships.

Reply:
> Yes, let's remove this invocation from here, but update the job to be more like the aws_eks_asset_exposure.json, where we would remove all the :HAS relationships, regarldess of lastupdated, and then create new :HAS relationships.

Consistency of "delete all then recreate" vs "delete only stale" is a problem, but this change is out of scope for this PR imo. What do you think, do you think this is blocking?